Cron/WebhookProcessor.php

```diff
@@ -65,7 +65,7 @@ public function execute()
         try {
             $this->doProcessWebhook();
         } catch (Exception $e) {
-            $this->adyenLogger->addAdyenNotification($e->getMessage() . "\n" . $e->getTraceAsString());
+            $this->adyenLogger->addAdyenWarning($e->getMessage() . "\n" . $e->getTraceAsString());
```

Comment:
Why this error is not part of the notification as it is related to the webhook process ?

Reply:
The idea is that any exception that is caught at this level means that the cron processing (which runs every minute) has terminated rather than skipping to the next notification. So I thought it makes sense to log this as a warning.

Reply:
ok thank you for the explanation :)